ext/TracyProfilerExt.jl

```diff
@@ -0,0 +1,13 @@
+module TracyProfilerExt
+
+using Tracy, TracyProfiler_jll
+
+function Tracy._capture(outfile::String, dummy::Symbol; gui::Bool = false, port::Int64 = 9001)
```

Comment:
This feels like it is bundling two different things in the same function. For example, the obligatory `outfile` is not used when `gui` is true. And the address argument is hardcoded for the GUI.

Wouldn't it make more sense to have one function per executable with arguments corresponding to the command line arguments that that executable takes?

Reply:
Yeah, I'm fine with that.